From: Yaroslav Halchenko <debian@onerussian.com>
Subject: Somewhat blind patch: removing 'MODLIBS' from link_args for exec's

 As Jakub guessed it seems to help to resolve not only -lffi issues with
 broken python3.2 pkg in debian sid, but with other build problems as well

 Needs more verification if this is a safe thing to do ;)

Origin: Debian
Last-Update: 2012-06-01

Index: mpi4py/setup.py
===================================================================
--- mpi4py.orig/setup.py	2021-08-14 18:27:08.409517252 +0200
+++ mpi4py/setup.py	2021-08-14 18:27:08.405517907 +0200
@@ -355,7 +355,7 @@
     for var in ('LIBDIR', 'LIBPL'):
         library_dirs += split_quoted(cfg_vars.get(var, ''))
     for var in ('LDFLAGS',
-                'LIBS', 'MODLIBS', 'SYSLIBS',
+                'LIBS', 'SYSLIBS',
                 'LDLAST'):
         link_args += split_quoted(cfg_vars.get(var, ''))
     exe.libraries += libraries
